web_fragments/tests/test_fragment.py

```diff
@@ -50,38 +87,7 @@ def validate_fragment(self, fragment=None, fragment_dict=None):
         self.assertEqual(fragment_dict['js_init_fn'], TEST_JS_INIT_FN)
         self.assertEqual(fragment_dict['js_init_version'], EXPECTED_JS_INIT_VERSION)
         self.assertEqual(fragment_dict['json_init_args'], TEST_JSON_INIT_ARGS)
-        self.assertEqual(fragment_dict['resources'], self.test_resources_dict())
-
-    def test_resources_dict(self):
-        """
-        Returns the expected test resources.
-        """
-        return [
-            {
-                'kind': 'text',
-                'data': TEST_CSS,
-                'mimetype': 'text/css',
-                'placement': 'head',
-            },
-            {
-                'kind': 'url',
-                'data': TEST_CSS_URL,
-                'mimetype': 'text/css',
-                'placement': 'head',
-            },
-            {
-                'kind': 'text',
-                'data': TEST_JS,
-                'mimetype': 'application/javascript',
-                'placement': 'foot',
-            },
-            {
-                'kind': 'url',
-                'data': TEST_JS_URL,
-                'mimetype': 'application/javascript',
-                'placement': 'foot',
-            },
-        ]
+        self.assertEqual(fragment_dict['resources'], EXPECTED_RESOURCES)
```

Comment:
This line is still using `self.assertEqual`. The rest of the file seems to have moved over to the `assert ==` syntax.